rust/rbac-registration/Cargo.toml

```diff
@@ -33,7 +33,6 @@ oid-registry = "0.7.1"
 thiserror = "2.0.11"
 
 c509-certificate = { version = "0.0.3", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "c509-certificate-v0.0.3" }
-pallas = { version = "0.33.0" }
 cbork-utils = { version = "0.0.2", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "cbork-utils-v0.0.2" }
-cardano-blockchain-types = { version = "0.0.5", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "cardano-blockchain-types-v0.0.5" }
-catalyst-types = { version = "0.0.5", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "catalyst-types-v0.0.5" }
+cardano-blockchain-types = { version = "0.1.5", git = "https://github.com/input-output-hk/catalyst-libs.git", branch = "wasm/cardano-bc-types" }
```

Comment:
Don't forget to use `tag` not `branch` after dependent crate is merged and tagged.
Make sure tag conforms to our standards.